announce rollout of dark theme (#3600)
Cumulocity-IoT/c8y-docs: content/change-logs/application-enablement/public-options-1-1-0-dark-theme-available-with-default-branding.md

---
date: 2025-08-06
title: Dark theme available for tenants with default branding
product_area: Application enablement & solutions
change_type:
  - value: change-inv-3bw8e
    label: Announcement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: ""
version: ""
---
A new version of the default branding will be rolled out to make the dark theme available for users.

> By default the light theme will still continue to be selected, but users receive the option to switch between themes via the right drawer.

Author: ```suggestion
By default, the light theme remains selected, but users can switch between themes via the right drawer.
```

Tenants having a customized branding will not be affected by this change.

> In case you would like to provide a dark theme as part of your custom branding, the [branding manager](/enterprise-tenant/customization/#branding) provides you the option to do so.

Author: ```suggestion
In case you would like to provide a dark theme as part of your custom branding, the [branding manager](/enterprise-tenant/customization/#branding) provides you with the option to do so.
```

> In case you would like to provide a dark theme as part of your custom branding, the [branding manager](/enterprise-tenant/customization/#branding) provides you with the option to do so.

Reviewer: ```suggestion
If you would like to provide a dark theme as part of your custom branding, you can do so via the [branding manager](/enterprise-tenant/customization/#branding).
```

> We highly recommend to upgrade any web apps based on the Web SDK to version 1022.8.0 or higher in order for the best compatibility and experience with the dark theme.

Reviewer: ```suggestion
We highly recommend upgrading any web apps based on the Web SDK to version 1022.8.0 or higher for best compatibility and experience with the dark theme.
```
